src/main.cpp

```diff
@@ -5944,7 +6001,12 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
       }
     else if (strCommand == NetMsgType::XPEDITEDBLK)  // BU
       {
-	HandleExpeditedBlock(vRecv,pfrom);
+	if (!HandleExpeditedBlock(vRecv,pfrom))
+        {
```

Comment:
I guess I should ask for Egyptian if I want Allman to happen here. I won't ask for Allman, because that'd get me Egyptian and now that most of the code is moving toward Allman...